docs/configuring-playbook-bot-postmoogle.md

```diff
@@ -21,6 +21,16 @@ matrix_bot_postmoogle_enabled: true
 
 # Generate a strong password here. Consider generating it with `pwgen -s 64 1`
 matrix_bot_postmoogle_password: PASSWORD_FOR_THE_BOT
+
+# Custom domains configuration for Postmoogle (optional)
+matrix_bot_postmoogle_domains:
+  - "other.domain"
+  - "{{ matrix_server_fqn_matrix }}"
+
+# SSL certificates configuration for custom domains
+# Optional configuration, required if custom domains for postmoogle are used
+matrix_ssl_additional_domains_to_obtain_certificates_for:
+  - "other.domain"
```

Comment:
This part is incorrect.

The `matrix_bot_postmoogle_domains`  list is already automatically injected into `matrix_ssl_domains_to_obtain_certificates_for` as seen here: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/704958f914ef88c90ffc88c9ca7a4df0b03d5774/group_vars/matrix_servers#L2742

There's no need to add them a second time by manually adding them as **additional** domains in `matrix_ssl_additional_domains_to_obtain_certificates_for`

Reply:
Without explicitly setting this variable `matrix_ssl_additional_domains_to_obtain_certificates_for` it wasn't working.